ui/static/js/app.js

```diff
@@ -390,13 +390,24 @@ function goToNext() {
 }
 
 function goToFeedOrFeeds() {
+    if (isEntry()) {
+        goToFeed();
+    } else {
+        goToPage('feeds');
+    }
+}
+
+function goToFeed() {
     if (isEntry()) {
         let feedAnchor = document.querySelector("span.entry-website a");
         if (feedAnchor !== null) {
             window.location.href = feedAnchor.href;
         }
     } else {
-        goToPage('feeds');
+        let currentItemFeed = document.querySelector(".current-item a[data-feed-link]");
+	if (currentItemFeed !== null) {
```

Comment:
It's cosmetic but the indentation should be consistent with the rest (4 spaces).

Reply:
Sorry about that! It looks like my editor had misconfigured tabbing. Thanks for pointing it out; it's fixed now.

I'll work on the keyboard shortcuts.